nexus/reconfigurator/planning/src/blueprint_builder.rs

```diff
@@ -494,7 +445,7 @@ impl<'a> BlueprintBuilder<'a> {
                 })
                 .collect();
 
-            (additions, removals)
+            dbg!((additions, removals))
```

Comment:
stray `dbg!`?

Reply:
Yes, nice catch. This was trying to figure out why https://github.com/oxidecomputer/omicron/pull/5583#discussion_r1572910123 was failing :sweat_smile: